### PR TITLE
Update Element Timing API pages

### PR DESCRIPTION
**Pull request by Elchi3**
### Description

This PR updates pages for the Element Timing API. The main page is https://developer.mozilla.org/en-US/docs/Web/API/PerformanceElementTiming. The overview page https://developer.mozilla.org/en-US/docs/Web/API/Element_timing_API is redirected to the main interface page per our plan to document Performance APIs as a single group.

### Motivation

https://github.com/openwebdocs/project/issues/62

### Additional details

- Adds missing docs for Element.elementTiming (reflection of the HTML attribute)
- Uses "type" and "buffered" in observe calls
- Removes unnecessary note box usage on the PerformanceElementTiming subpages. (prime example how I think notes should not be used: This shouldn't be note-worthy, this is just normal explanatory text about what the example code does).
- Adds cross-origin info to the renderTime page.
 
### Related issues and pull requests

None.

### CODE REVIEW

File: files/en-us/web/api/element/elementtiming/index.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+### Logging the value of `elementTiming`
+
+In this example an {{HTMLElement("image")}} element is set to being observed by adding the [`elementtiming`](/en-US/docs/Web/HTML/Attributes/elementtiming) attribute.
+
+```html
+<img
+  src="image.jpg"
+  alt="a nice image"
+  elementtiming="big-image"
```
Comment (estelle) [inline files/en-us/web/api/element/elementtiming/index.md:26]: should we add this attribute to the [`<imag>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img) element? It's on the sidebar but not in the text.

File: files/en-us/web/api/performanceelementtiming/element/index.md [hunk 2]
```diff
@@ -14,16 +14,17 @@ browser-compat: api.PerformanceElementTiming.element
...
-An {{domxref("Element")}}.
+An {{domxref("Element")}} or {{jsxref("null")}} if the element is a shadow DOM element.
 
 ## Examples
 
-In this example calling `entry.element` will log to the console
-`<img src="image.jpg" alt="a nice image" elementtiming="big-image">`.
+### Logging the observed element
+
+In this example an {{HTMLElement("image")}} element is being observed by adding the [`elementtiming`](/en-US/docs/Web/HTML/Attributes/elementtiming) attribute. A {{domxref("PerformanceObserver")}} is registered to get all performance entries of type `"element"` and the `buffered` flag is used to access data from before observer creation. The DOM element that is observed is logged to the console.
```
Comment (estelle) [inline files/en-us/web/api/performanceelementtiming/element/index.md:27]:
```suggestion
In this example an {{HTMLElement("img")}} element is being observed by adding the [`elementtiming`](/en-US/docs/Web/HTML/Attributes/elementtiming) attribute. A {{domxref("PerformanceObserver")}} is registered to get all performance entries of type `"element"` and the `buffered` flag is used to access data from before observer creation. The DOM element that is observed is logged to the console.
```

File: files/en-us/web/api/performanceelementtiming/id/index.md [hunk 3]
```diff
@@ -22,7 +22,9 @@ A string.
 
 ## Examples
 
-In this example calling `entry.id` will log to the console `myImage`, this being the {{htmlattrxref("id")}} of the image element.
+### Using `id`
+
+In this example an {{HTMLElement("image")}} element is being observed by adding the [`elementtiming`](/en-US/docs/Web/HTML/Attributes/elementtiming) attribute. A {{domxref("PerformanceObserver")}} is registered to get all performance entries of type `"element"` and the `buffered` flag is used to access data from before observer creation. It will log `myImage` to the console, this being the {{htmlattrxref("id")}} of the image element.
```
Comment (estelle) [inline files/en-us/web/api/performanceelementtiming/id/index.md:27]:
```suggestion
In this example an {{HTMLElement("img")}} element is being observed by adding the [`elementtiming`](/en-US/docs/Web/HTML/Attributes/elementtiming) attribute. A {{domxref("PerformanceObserver")}} is registered to get all performance entries of type `"element"` and the `buffered` flag is used to access data from before observer creation. It will log `myImage` to the console, this being the {{htmlattrxref("id")}} of the image element.
```

File: files/en-us/web/api/performanceelementtiming/identifier/index.md [hunk 4]
```diff
@@ -22,7 +22,9 @@ A string.
 
 ## Examples
 
-In this example the value of [`elementtiming`](/en-US/docs/Web/HTML/Attributes/elementtiming) is `big-image`. Calling `entry.identifier` therefore returns the string `big-image`.
+### Using `identifier`
+
+In this example an {{HTMLElement("image")}} element is being observed by adding the [`elementtiming`](/en-US/docs/Web/HTML/Attributes/elementtiming) attribute. A {{domxref("PerformanceObserver")}} is registered to get all performance entries of type `"element"` and the `buffered` flag is used to access data from before observer creation. The value of [`elementtiming`](/en-US/docs/Web/HTML/Attributes/elementtiming) is `big-image`. Calling `entry.identifier` therefore returns the string `big-image`.
```
Comment (estelle) [inline files/en-us/web/api/performanceelementtiming/identifier/index.md:27]:
```suggestion
In this example an {{HTMLElement("img")}} element is being observed by adding the [`elementtiming`](/en-US/docs/Web/HTML/Attributes/elementtiming) attribute. A {{domxref("PerformanceObserver")}} is registered to get all performance entries of type `"element"` and the `buffered` flag is used to access data from before observer creation. The value of [`elementtiming`](/en-US/docs/Web/HTML/Attributes/elementtiming) is `big-image`. Calling `entry.identifier` therefore returns the string `big-image`.
```

File: files/en-us/web/api/performanceelementtiming/index.md [hunk 5]
```diff
@@ -13,12 +13,41 @@ browser-compat: api.PerformanceElementTiming
...
+
+## Description
+
+The aim of the Element Timing API is to give web developers or analytics tools the ability to measure rendering timestamps of critical elements on a page.
+
+The API supports timing information on the following elements:
+
+- {{htmlelement("img")}} elements,
+- {{SVGElement("image")}} elements inside an {{SVGElement("svg")}},
+- poster images of {{htmlelement("video")}} elements,
```
Comment (estelle) [inline files/en-us/web/api/performanceelementtiming/index.md:26]:
```suggestion
- [poster](/en-US/docs/Web/HTML/Element/video#attr-poster) images of {{htmlelement("video")}} elements,
```

File: files/en-us/web/api/performanceelementtiming/index.md [hunk 6]
```diff
@@ -13,12 +13,41 @@ browser-compat: api.PerformanceElementTiming
...
+- elements which have a {{cssxref("background-image")}},
+- and groups of text nodes, such as a {{htmlelement("p")}}.
```
Comment (estelle) [inline files/en-us/web/api/performanceelementtiming/index.md:28]:
```suggestion
- elements which have a {{cssxref("background-image")}}, and
- groups of text nodes, such as a {{htmlelement("p")}}.
```

File: files/en-us/web/api/performanceelementtiming/intersectionrect/index.md [hunk 7]
```diff
@@ -24,7 +24,9 @@ For display images this is the display rectangle of the image within the viewpor
 
 ## Examples
 
-In this example calling `entry.intersectionRect` returns a {{domxref("DOMRectReadOnly")}} object with the display rectangle of the image.
+### Logging `intersectionRect`
+
+In this example an {{HTMLElement("image")}} element is being observed by adding the [`elementtiming`](/en-US/docs/Web/HTML/Attributes/elementtiming) attribute. A {{domxref("PerformanceObserver")}} is registered to get all performance entries of type `"element"` and the `buffered` flag is used to access data from before observer creation. Calling `entry.intersectionRect` returns a {{domxref("DOMRectReadOnly")}} object with the display rectangle of the image.
```
Comment (estelle) [inline files/en-us/web/api/performanceelementtiming/intersectionrect/index.md:29]:
```suggestion
In this example an {{HTMLElement("img")}} element is being observed by adding the [`elementtiming`](/en-US/docs/Web/HTML/Attributes/elementtiming) attribute. A {{domxref("PerformanceObserver")}} is registered to get all performance entries of type `"element"` and the `buffered` flag is used to access data from before observer creation. Calling `entry.intersectionRect` returns a {{domxref("DOMRectReadOnly")}} object with the display rectangle of the image.
```

File: files/en-us/web/api/performanceelementtiming/loadtime/index.md [hunk 8]
```diff
@@ -14,15 +14,17 @@ browser-compat: api.PerformanceElementTiming.loadTime
...
 
-A {{domxref("DOMHighResTimeStamp")}} with the loadTime of the element.
+A {{domxref("DOMHighResTimeStamp")}} with the `loadTime` of the element. Always `0` for text.
 
 ## Examples
 
-In this example calling `entry.loadTime` returns the loadTime of the image element.
+### Logging `loadTime`
+
+In this example an {{HTMLElement("image")}} element is being observed by adding the [`elementtiming`](/en-US/docs/Web/HTML/Attributes/elementtiming) attribute. A {{domxref("PerformanceObserver")}} is registered to get all performance entries of type `"element"` and the `buffered` flag is used to access data from before observer creation. Calling `entry.loadTime` returns the loadTime of the image element.
```
Comment (estelle) [inline files/en-us/web/api/performanceelementtiming/loadtime/index.md:27]:
```suggestion
In this example an {{HTMLElement("img")}} element is being observed by adding the [`elementtiming`](/en-US/docs/Web/HTML/Attributes/elementtiming) attribute. A {{domxref("PerformanceObserver")}} is registered to get all performance entries of type `"element"`. The `buffered` flag is used to access data from before the observer was created. Calling `entry.loadTime` returns the loadTime of the image element.
```

Comment (estelle) [reply files/en-us/web/api/performanceelementtiming/loadtime/index.md:27]: not sure if my edit is correct. If it is, then all occurrences...

File: files/en-us/web/api/performanceelementtiming/url/index.md [hunk 9]
```diff
@@ -18,11 +18,13 @@ The **`url`** read-only property of the {{domxref("PerformanceElementTiming")}}
...
 
-A string which is the initial URL of the resources request for images or 0 for text.
+A string which is the initial URL of the resources request for images or `0` for text.
 
 ## Examples
 
-In this example calling `entry.url` returns `https://example.com/image.jpg`.
+### Logging `url`
+
+In this example an {{HTMLElement("image")}} element is being observed by adding the [`elementtiming`](/en-US/docs/Web/HTML/Attributes/elementtiming) attribute. A {{domxref("PerformanceObserver")}} is registered to get all performance entries of type `"element"` and the `buffered` flag is used to access data from before observer creation. Calling `entry.url` returns `https://example.com/image.jpg`.
```
Comment (estelle) [inline files/en-us/web/api/performanceelementtiming/url/index.md:27]:
```suggestion
In this example an {{HTMLElement("img")}} element is being observed by adding the [`elementtiming`](/en-US/docs/Web/HTML/Attributes/elementtiming) attribute. A {{domxref("PerformanceObserver")}} is registered to get all performance entries of type `"element"` and the `buffered` flag is used to access data from before observer creation. Calling `entry.url` returns `https://example.com/image.jpg`.
```

**Review by estelle — COMMENTED** [review]
two questions. one issue (repeated multiple times)

File: files/en-us/web/api/element/elementtiming/index.md [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+
+## Value
+
+A string.
+
+## Examples
+
+### Logging the value of `elementTiming`
+
+In this example an {{HTMLElement("image")}} element is set to being observed by adding the [`elementtiming`](/en-US/docs/Web/HTML/Attributes/elementtiming) attribute.
```
Comment (estelle) [inline files/en-us/web/api/element/elementtiming/index.md:20]:
```suggestion
In this example, adding the [`elementtiming`](/en-US/docs/Web/HTML/Attributes/elementtiming) attribute to the {{HTMLElement("img")}} element sets the image to be observed.
```

File: files/en-us/web/api/element/elementtiming/index.md [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+
+```html
+<img
+  src="image.jpg"
+  alt="a nice image"
+  elementtiming="big-image"
+  id="myImage" />
+```
+
+You can get the string provided to the `elementtiming` HTML attribute by calling `el.elementTiming`.
```
Comment (estelle) [inline files/en-us/web/api/element/elementtiming/index.md:30]:
```suggestion
You can get the string value of the `elementtiming` HTML attribute by calling `el.elementTiming`.
```

File: files/en-us/web/api/element/elementtiming/index.md (hunk 10 above)
Comment (estelle) [reply files/en-us/web/api/element/elementtiming/index.md:20]: alternatively, 

```
In this example, adding the [`elementtiming`](/en-US/docs/Web/HTML/Attributes/elementtiming) attribute to the {{HTMLElement("img")}} element sets the image to be observed by the Element Timing API.
```

File: files/en-us/web/api/element/elementtiming/index.md (hunk 1 above)
Comment (Elchi3) [reply files/en-us/web/api/element/elementtiming/index.md:26]: Added in https://github.com/mdn/content/pull/22846/commits/a28da1661a63cc92330f8bcf281c7aef14269d42

**Comment by Elchi3** [comment]
I've addressed the review comments and resolved the merge conflicts.

**Comment by bsmth** [comment]
Taking a look now 👀

File: files/en-us/web/api/element/elementtiming/index.md [hunk 12]
```diff
@@ -0,0 +1,50 @@
+---
+title: Element.elementTiming
+slug: Web/API/Element/elementTiming
+page-type: web-api-instance-property
+browser-compat: api.Element.elementTiming
+---
+
+{{DefaultAPISidebar("DOM")}}
+
+The **`elementTiming`** property of the {{domxref("Element")}} interface reflects the value of the [`elementtiming`](/en-US/docs/Web/HTML/Attributes/elementtiming) attribute which marks an element for observation in the {{domxref("PerformanceElementTiming")}} API.
```
Comment (bsmth) [inline files/en-us/web/api/element/elementtiming/index.md:10]: This is a bit hard to parse, should we try to restructure? I'm not sure if this is an improvement:

```suggestion
The **`elementTiming`** property of the {{domxref("Element")}} interface identifies elements for observation in the {{domxref("PerformanceElementTiming")}} API. The `elementTiming` property reflects the value of the [`elementtiming`](/en-US/docs/Web/HTML/Attributes/elementtiming) attribute.
```

File: files/en-us/web/api/element/index.md [hunk 13]
```diff
@@ -47,6 +47,8 @@ _`Element` inherits properties from its parent interface, {{DOMxRef("Node")}}, a
   - : Returns a number representing the width of the top border of the element.
 - {{DOMxRef("Element.clientWidth")}} {{ReadOnlyInline}}
   - : Returns a number representing the inner width of the element.
+- {{DOMxRef("Element.elementTiming")}}
+  - : A string reflecting the [`elementtiming`](/en-US/docs/Web/HTML/Attributes/elementtiming) attribute which marks an element for observation in the {{domxref("PerformanceElementTiming")}} API.
```
Comment (bsmth) [inline files/en-us/web/api/element/index.md:51]: I have a similar feeling about "marks an element" phrasing, would it be clearer to say it's "an identifier" or "identifies an element for observation"?

File: files/en-us/web/api/element/elementtiming/index.md (hunk 12 above)
Comment (Elchi3) [reply files/en-us/web/api/element/elementtiming/index.md:10]: Yes, I think this is better. I don't know if we have a standard wording for HTML content attribute reflection and what it means. Maybe something to standardize some day.

File: files/en-us/web/api/element/index.md (hunk 13 above)
Comment (Elchi3) [reply files/en-us/web/api/element/index.md:51]:
```suggestion
  - : A string reflecting the [`elementtiming`](/en-US/docs/Web/HTML/Attributes/elementtiming) attribute which identifies an element for observation in the {{domxref("PerformanceElementTiming")}} API.
```

File: files/en-us/web/api/performanceelementtiming/id/index.md [hunk 14]
```diff
@@ -22,7 +22,9 @@ A string.
 
 ## Examples
 
-In this example calling `entry.id` will log to the console `myImage`, this being the {{htmlattrxref("id")}} of the image element.
+### Using `id`
+
+In this example an {{HTMLElement("img")}} element is being observed by adding the [`elementtiming`](/en-US/docs/Web/HTML/Attributes/elementtiming) attribute. A {{domxref("PerformanceObserver")}} is registered to get all performance entries of type `"element"` and the `buffered` flag is used to access data from before observer creation. It will log `myImage` to the console, this being the {{htmlattrxref("id")}} of the image element.
```
Comment (bsmth) [inline files/en-us/web/api/performanceelementtiming/id/index.md:27]: Nitpicking, I guess adding `elementtiming` attr doesn't start observation but acts as an identifier:

```suggestion
In this example, an {{HTMLElement("img")}} element is given an [`elementtiming`](/en-US/docs/Web/HTML/Attributes/elementtiming) attribute so it can be identified during performance observation. A {{domxref("PerformanceObserver")}} is registered to get all performance entries of type `"element"` and the `buffered` flag is used to access data from before observer creation. It will log `myImage` to the console, which is the {{htmlattrxref("id")}} attribute of the image element.
```

File: files/en-us/web/api/performanceelementtiming/identifier/index.md [hunk 15]
```diff
@@ -22,7 +22,9 @@ A string.
 
 ## Examples
 
-In this example the value of [`elementtiming`](/en-US/docs/Web/HTML/Attributes/elementtiming) is `big-image`. Calling `entry.identifier` therefore returns the string `big-image`.
+### Using `identifier`
+
+In this example an {{HTMLElement("img")}} element is being observed by adding the [`elementtiming`](/en-US/docs/Web/HTML/Attributes/elementtiming) attribute. A {{domxref("PerformanceObserver")}} is registered to get all performance entries of type `"element"` and the `buffered` flag is used to access data from before observer creation. The value of [`elementtiming`](/en-US/docs/Web/HTML/Attributes/elementtiming) is `big-image`. Calling `entry.identifier` therefore returns the string `big-image`.
```
Comment (bsmth) [inline files/en-us/web/api/performanceelementtiming/identifier/index.md:27]: Same comment as above, but I can see there's a need to disambiguate with `identifier` in this case. The part I think that's unclear for me is whether "is being observed by adding the `elementtiming` attribute" is true or not. Does adding this attribute alone without creating an observer do anything?

File: files/en-us/web/api/performanceelementtiming/id/index.md (hunk 14 above)
Comment (bsmth) [reply files/en-us/web/api/performanceelementtiming/id/index.md:27]: Correct me if I'm wrong here :)

File: files/en-us/web/api/performanceelementtiming/loadtime/index.md [hunk 16]
```diff
@@ -14,15 +14,17 @@ browser-compat: api.PerformanceElementTiming.loadTime
 
 {{APIRef("Performance API")}}{{SeeCompatTable}}
 
-The **`loadTime`** read-only property of the {{domxref("PerformanceElementTiming")}} interface always returns 0 for text. For images it returns the time which is the latest between the time the image resource is loaded and the time it is attached to the element.
+The **`loadTime`** read-only property of the {{domxref("PerformanceElementTiming")}} interface always returns `0` for text. For images it returns the time which is the latest between the time the image resource is loaded and the time it is attached to the element.
```
Comment (bsmth) [inline files/en-us/web/api/performanceelementtiming/loadtime/index.md:17]: >For images it returns the time which is the latest between the time the image resource is loaded and the time it is attached to the element.

I don't follow this one, what's 'attaching to the element' in this context? 🤔

File: files/en-us/web/api/performanceelementtiming/id/index.md (hunk 14 above)
Comment (Elchi3) [reply files/en-us/web/api/performanceelementtiming/id/index.md:27]: hm, I think the `elementtiming` attribute alone does in fact signal to the browser to store performance data about the element even if there is no code yet that accesses this data using a `PerformanceObserver`. The `buffered` flag is used to access data from before observer creation.

File: files/en-us/web/api/performanceelementtiming/loadtime/index.md (hunk 16 above)
Comment (Elchi3) [reply files/en-us/web/api/performanceelementtiming/loadtime/index.md:17]: I think it means that the image resource the browser loaded need to be attached to the element where it should appear. So, browser loads image resource img.png at a URL and then attaches that resource to where it was requested in the DOM (maybe an  or other element where e.g. CSS background-image was used).

File: files/en-us/web/api/performanceelementtiming/identifier/index.md (hunk 15 above)
Comment (Elchi3) [reply files/en-us/web/api/performanceelementtiming/identifier/index.md:27]: Yeah, adding this attribute alone should already do something, I think. Observers can even access data from before observer creation with their "buffered" flag, so `elementtiming` is the key thing to tell browsers to record data.

**Review by bsmth — APPROVED** [review]
Thanks @Elchi3 👍🏻
